frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1107,7 +1099,7 @@ export const surveyLogic = kea<surveyLogicType>([
                 return survey.questions.length > 1
             },
         ],
-        showSurveyRepeatSchedule: [(s) => [s.schedule], (schedule: ScheduleType) => schedule == 'recurring'],
+        showSurveyRepeatSchedule: [(s) => [s.survey], (survey: Survey) => survey.schedule === 'recurring'],
```

Comment:
can we reuse the enum for `recurring` as well?

Reply:
changed on[ other places too](https://github.com/PostHog/posthog/pull/28896/commits/493fd2d4cc0b0a267ed606010682c849a8d97f60)